tests/backup/backup_kubevirt_test.go

```diff
@@ -2150,3 +2150,486 @@ var _ = Describe("{KubevirtVMRestoreWithAfterChangingVMConfig}", func() {
 		CleanupCloudSettingsAndClusters(backupLocationMap, cloudCredName, cloudCredUID, ctx)
 	})
 })
+
+// This testcase verifies default backup & restore with both Kubevirt VMs and non-kubevirt resources
+var _ = Describe("{DefaultBackupRestoreWithKubevirtAndNonKubevirtNS}", func() {
+	var (
+		backupName         string
+		restoreName        string
+		scheduleBackupName string
+		backupNames        []string
+		//restoreNames               []string
+		//replaceRestoreNames        []string
+		scheduleNames              []string
+		scheduledAppContexts       []*scheduler.Context
+		singleScheduledAppContexts []*scheduler.Context
+		multiScheduledAppContexts  []*scheduler.Context
+		sourceClusterUID           string
+		cloudCredName              string
+		cloudCredUID               string
+		backupLocationUID          string
+		backupLocationName         string
+		backupLocationMap          map[string]string
+		labelSelectors             map[string]string
+		//namespaceMapping           map[string]string
+		providers          []string
+		numOfDeployments   int
+		appNamespaces      []string
+		schPolicyUid       string
+		periodicPolicyName string
+		preRuleName        string
+		postRuleName       string
+		preRuleUid         string
+		postRuleUid        string
+		preRuleNames       []string
+		postRuleNames      []string
+		allVMs             []kubevirtv1.VirtualMachine
+		allVMNames         []string
+		preRuleList        []*api.RulesInfo_RuleItem
+		postRuleList       []*api.RulesInfo_RuleItem
+		//backupNameToAppContextMap  map[string][]*scheduler.Context
+		//restoreNameBackupNameMap   map[string]string
+	)
+
+	backupLocationMap = make(map[string]string)
+	backupNames = make([]string, 0)
+	//restoreNames = make([]string, 0)
+	//replaceRestoreNames = make([]string, 0)
+	labelSelectors = make(map[string]string)
+	appNamespaces = make([]string, 0)
+	backupNamespaceMap := make(map[string]string)
+	//restoreNameBackupNameMap = make(map[string]string)
+	//backupNameToAppContextMap = make(map[string][]*scheduler.Context)
+	periodicPolicyName = fmt.Sprintf("%s-%s", "periodic", RandomString(6))
+	backupDriver := Inst().Backup
+
+	JustBeforeEach(func() {
+		StartPxBackupTorpedoTest("DefaultBackupRestoreWithKubevirtAndNonKubevirtNS", "Verify default backup & restore with both Kubevirt and Non-Kubevirt namespaces", nil, 93006, Vpinisetti, Q1FY25)
+		numOfDeployments = Inst().GlobalScaleFactor
+		providers = GetBackupProviders()
+		appList := Inst().AppList
+		defer func() {
+			Inst().AppList = appList
+		}()
+		Inst().AppList = []string{"mysql-backup", "kubevirt-cirros-cd-with-pvc"}
+		log.InfoD("Deploying all provided applications in a single namespace")
+		for i := 0; i < numOfDeployments; i++ {
+			taskName := fmt.Sprintf("%s-%d-%d", TaskNamePrefix, 93006, i)
```

Comment:
I think the TestRailID is added by default; StartPxBackupTorpedoTest handles it.